docs/_embed/quick-account-creation.md

```diff
@@ -43,14 +43,28 @@ If you have an existing Full Zapier Experience or Zap templates element embed, y
 
 ## Add support to the Partner API
 
-1. Retrieve your integration's name from the "title" field returned from the [`/apps` endpoint](https://platform.zapier.com/embed/partner-api#get-v1apps).
-2. Redirect users to the acknowledgement page URL below:
-`https://zapier.com/partner/acknowledgment?sign-up-last-name=<sign-up-last-name>&next=%2Fapp%2Fdashboard&type=quac&name=<Partner Name>&sign-up-first-name=<sign-up=first-name>&sign-up-email=<sign-up-email>`
-3. Replace the following query parameter placeholders in the URL, using the "title" value from step 1 for the "name" parameter:
-- name=`<partner-name>`
+1. Use the URL below to initiate Quick Account Creation, then redirect users into the Editor with the Zap template set in the parameters:
+`https://zapier.com/webintent/create-zap?
+template=<zap-template-id>&
+utm_source=partner&
+utm_medium=embed&
+utm_campaign=partner_api&
+utm_content=partner_quick_account_creation&
+entry-point-location=partner_embed&
+referer=<referer>&
+referrer=<referrer>&
+sign-up-first-name=<sign-up-first-name>&
+sign-up-last-name=<sign-up-last-name>&
+sign-up-email=<sign-up-email>&
+client-id=<client-id>`
+2. Replace the following query parameter placeholders in the URL:
+- client-id=`<client-id>`
+- template=`<zap-template-id>`
 - sign-up-first-name=`<sign-up-first-name>`
 - sign-up-last-name=`<sign-up-last-name>`
 - sign-up-email=`<sign-up-email>`
+- referer=`<referer>`
+- referrer=`<referrer>`
```

Comment:
The formatting feels a little hard to follow here

![image](https://github.com/zapier/visual-builder/assets/176815/b6aeddac-c69f-4630-8ec5-cf6f452d3fc2)

and the URL has spaces after each `&` and `?`

Reply:
oh weird about the spaces - good catch.

Reply:
Hope this is better! Matched it with the formatting of URLs & params from the Partner API doc. Thanks for keeping me honest with the formatting!

Reply:
I appreciate the table! Makes it much easier to follow imo